addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -137,7 +137,7 @@
                                 <ImportoBollo t-esc="format_numbers(record.l10n_it_stamp_duty)"/>
                             </DatiBollo>
                         </DatiGeneraliDocumento>
-                        <DatiDDT t-if="record.l10n_it_ddt_id">
+                        <DatiDDT t-if="record.l10n_it_ddt_id" name="DDT">
```

Comment:
Is this name necessary (problem in stable) as maybe there is already just one DatiDDT tag.

Reply:
it done.